test/integration_test.sh

```diff
@@ -19,6 +35,10 @@ curl -s localhost:8080/static2/ducks.jpg > test_output_static2_jpg
 curl -s localhost:8080/static2/giphy.gif > test_output_static2_gif
 curl -s localhost:8080/static2/mario.jpeg > test_output_static2_jpeg
 
+#proxy test
+curl -i -s localhost:8080/test > test_proxy_echo2
```

Comment:
Hm, my understanding is that all the integration tests go together in one large file? That's what we did for ours so this is okay imo unless you'd prefer i separate it